keps/sig-contributor-experience/0034-artifact-management.md

```diff
@@ -0,0 +1,124 @@
+
+1. **Fill out the "overview" sections.**
+  This includes the Summary and Motivation sections.
+  These should be easy if you've preflighted the idea of the KEP with the appropriate SIG.
+1. **Create a PR.**
+  Assign it to folks in the SIG that are sponsoring this process.
+1. **Merge early.**
+  Avoid getting hung up on specific details and instead aim to get the goal of the KEP merged quickly.
+  The best way to do this is to just start with the "Overview" sections and fill out details incrementally in follow on PRs.
+  View anything marked as a `provisional` as a working document and subject to change.
+  Aim for single topic PRs to keep discussions focused.
+  If you disagree with what is already in a document, open a new PR with suggested changes.
+
+The canonical place for the latest set of instructions (and the likely source of this file) is [here](/keps/0000-kep-template.md).
+
+The `Metadata` section above is intended to support the creation of tooling around the KEP process.
+This will be a YAML section that is fenced as a code block.
+See the KEP process for details on each of these items.
+
+## Table of Contents
+
+A table of contents is helpful for quickly jumping to sections of a KEP and for highlighting any additional information provided beyond the standard KEP template.
+[Tools for generating][] a table of contents from markdown are available.
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [User Stories [optional]](#user-stories-optional)
+      * [Story 1](#story-1)
+      * [Story 2](#story-2)
+    * [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+    * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+* [Drawbacks [optional]](#drawbacks-optional)
+* [Alternatives [optional]](#alternatives-optional)
+
+[Tools for generating]: https://github.com/ekalinin/github-markdown-toc
+
+## Summary
+This document describes how official artifacts (Container Images, Binaries) for the Kubernetes
+project are managed and distributed.
+
+
+## Motivation
+
+The motivation for this KEP is to describe a process by which artifacts (container images, binaries)
+can be distributed by the community. Currently the process by which images is both ad-hoc in nature
+and limited to an arbitrary set of people who have the keys to the relevant repositories. Standardize
+access will ensure that people around the world have access to the same artifacts by the same names
+and that anyone in the project is capable (if given the right authority) to distribute images.
+
+### Goals
+
+The goals of this process are to enable:
+  * Anyone in the community (with the right permissions) to manage the distribution of Kubernetes images and binaries.
```

Comment:
This is a little confusing for me based on the non-goals.
Do you expect to document the permissions to manage the distribution here?
Is this proposal to serve artifacts across the globe or to manage them?

Reply:
Nope, out of scope. This is just to imply that anyone in the community is capable of it, as opposed to currently where the keys are held by a random few.